math.c

```diff
@@ -457,6 +457,33 @@ math_exp(VALUE unused_obj, VALUE x)
     return DBL2NUM(exp(Get_Double(x)));
 }
 
+/*
+ *  call-seq:
+ *    Math.expm1(x) -> float
+ *
+ *  Returns +e+ raised to the +x+ power minus 1.
```

Comment:
The precedence is not indicated very well with the texts, this could be `Math.exp(x) - 1` or `Math.exp(x - 1)`

The other,  "Returns the base E [logarithm](https://en.wikipedia.org/wiki/Logarithm) of `x` + 1." is more confusing since operator `+` is generally lower precedence.